src/components/CodeBlock.tsx

```diff
@@ -4,7 +4,7 @@ import theme from 'prism-react-renderer/themes/nightOwl'
 import * as React from 'react'
 import { LiveEditor, LiveError, LivePreview, LiveProvider } from 'react-live'
 
-import { styled } from '../stitches.config'
+import { styled } from '../../stitches.config'
```

Comment:
Maybe adding aliases to avoid this (like it Gary did on components)?

Very minor though

Reply:
Yeah I tried to but couldn't get it to work, will set this up in a separate PR 👍